### DISCUSSION TITLE
Import sessions from Session Buddy is not working

### DISCUSSION THREAD

**Discussion by ceasrj** [sienori/Tab-Session-Manager#1351] [Q&A]
Hi!
I'm trying to import a Session Buddy json and this error occurs:

![image](https://github.com/sienori/Tab-Session-Manager/assets/12467445/8f00d3f2-72a9-4edb-9c37-8d40a4e7716e)

Maybe the exported json structure has changed

Session Buddy version: v4.0.4

**Comment by Fizik79** [comment]
Also interested in this question is how you can import sessions from Session Buddy. After approximately version 4, Session Buddy changed the way it exports sessions - in new versions there are no options like "Sessions" and "Windows" and the export is made to a json file with a new structure. I would really like to import sessions; I have more than one hundred and fifty of them in Session Buddy. Does anyone have a solution?
